Module-3-Assignment/AccountContacts.cls

```diff
@@ -0,0 +1,28 @@
+/**
+* Class to set contactIds of Account
+* */
+public class AccountContacts {
+
+   /*
+    * Used to set ContactIds to contact list if null otherwise set with contactId
+    **/
+   public static String setContacts(Id contactId) {
+       Contact con = [SELECT AccountId from Contact Where Id = :contactID] ;
+       Account acc = [SELECT Id, ContactIds__c from Account Where Id = :con.AccountId];
+       if(acc.ContactIDs__c == null) {
+           List<Contact> contacts = [Select Id from Contact Where AccountID = :acc.Id];
+           String str = '';
+           for (Contact cont : contacts) {
+               str = str + cont.Id + ',';
```

Comment:
Change to normal for loop this will add comma at the end

Reply:
Done

Reply:
Do this with for each loop also

Reply:
Done